Firestore/Example/Tests/Util/FSTIntegrationTestCase.mm

```diff
@@ -170,6 +180,13 @@ + (NSString *)projectID {
   return defaultProjectId;
 }
 
++ (bool)isRunningAgainstEmulator {
+  if (!defaultProjectId) {
```

Comment:
Can you add a comment on why this lazy initialization is necessary?

Reply:
Done.